### ISSUE TITLE
Retraining Inception3 Model on new classification task error

### ISSUE THREAD

**Issue by sree369nidhi** [tensorflow/hub#370]
I0930 03:32:10.317262 22676 retrain.py:1054] 2019-09-30 03:32:10.317262: Step 3999: Validation accuracy = 93.0% (N=100)
2019-09-30 03:32:49.047830: W tensorflow/core/graph/graph_constructor.cc:1352] Importing a graph with a lower producer version 29 into an existing graph with producer version 38. Shape inference will have run different parts of the graph with different producer versions.
INFO:tensorflow:Saver not created because there are no variables in the graph to restore
I0930 03:32:56.514015 22676 saver.py:1499] Saver not created because there are no variables in the graph to restore
WARNING:tensorflow:From C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\training\saver.py:1276: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
W0930 03:32:57.629627 22676 deprecation.py:323] From C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\training\saver.py:1276: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Restoring parameters from /tmp/_retrain_checkpoint
I0930 03:32:57.637624 22676 saver.py:1280] Restoring parameters from /tmp/_retrain_checkpoint
INFO:tensorflow:Final test accuracy = 91.0% (N=2030)
I0930 03:33:00.087152 22676 retrain.py:792] Final test accuracy = 91.0% (N=2030)
INFO:tensorflow:Save final result to : /tf_files/dog_retrained_mobilenet10_224.pb
I0930 03:33:00.374989 22676 retrain.py:1082] Save final result to : /tf_files/dog_retrained_mobilenet10_224.pb
2019-09-30 03:33:00.713713: W tensorflow/core/graph/graph_constructor.cc:1352] Importing a graph with a lower producer version 29 into an existing graph with producer version 38. Shape inference will have run different parts of the graph with different producer versions.
INFO:tensorflow:Saver not created because there are no variables in the graph to restore
I0930 03:33:05.988166 22676 saver.py:1499] Saver not created because there are no variables in the graph to restore
INFO:tensorflow:Restoring parameters from /tmp/_retrain_checkpoint
I0930 03:33:07.191204 22676 saver.py:1280] Restoring parameters from /tmp/_retrain_checkpoint
WARNING:tensorflow:From tensorflow/examples/image_retraining/retrain.py:840: convert_variables_to_constants (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.graph_util.convert_variables_to_constants`
W0930 03:33:08.489981 22676 deprecation.py:323] From tensorflow/examples/image_retraining/retrain.py:840: convert_variables_to_constants (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.graph_util.convert_variables_to_constants`
WARNING:tensorflow:From C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\graph_util_impl.py:270: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.graph_util.extract_sub_graph`
W0930 03:33:08.489981 22676 deprecation.py:323] From C:\ProgramData\Anaconda3\lib\site-packages\tensorflow\python\framework\graph_util_impl.py:270: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.compat.v1.graph_util.extract_sub_graph`
INFO:tensorflow:Froze 378 variables.
I0930 03:33:09.614906 22676 graph_util_impl.py:311] Froze 378 variables.
INFO:tensorflow:Converted 378 variables to const ops.
I0930 03:33:10.755422 22676 graph_util_impl.py:364] Converted 378 variables to const ops.

**Comment by rmothukuru** [comment]
Please provide details about what platform you are using (operating system, architecture). Also include your TensorFlow and Tensorflow Hub versions

Make sure you also include the exact command if possible to produce the output included in your test case. If you are unclear what to include see the issue template displayed in [the Github new issue template](https://github.com/tensorflow/tensorflow/issues/new/choose).

We ask for this in the issue submission template, because it is really difficult to help without that information. Thanks!

**Comment by sree369nidhi** [comment]
thank you for the quick response but I got expected output since I am using windows those files got saved in C drive(windows drive) but the tutorial in TensorFlow was on Linux so got confused...

**Comment by rmothukuru** [comment]
Closing the issue as it has been resolved.
